cmd/logger.go

```diff
@@ -30,6 +34,15 @@ import (
 
 var log = NewLogger()
 
+type logEntry struct {
+	Level   string `json:"level"`
+	Source  string `json:"source"`
+	Cause   string `json:"cause"`
+	Message string `json:"message"`
+	Time    string `json:"time"`
+	Stack   string `json:"stack"`
```

Comment:
If `Stack` is stacktrace, array of string is better than single string.

Reply:
Will change it

Reply:
done